### PR TITLE
aii-opennebula: Include VM placement algorithm support

### LINKED ISSUE #234: aii-opennebula : Include scheduling support
We use scheduling feature of opennebula.
@alvarosimon is testing the proposed implementation.
Reference : http://docs.opennebula.org/5.2/operation/references/template.html#placement-section

### PR DESCRIPTION
**Pull request by alvarosimon**
* Fix #234 : Include scheduling support

### CODE REVIEW

**Comment by alvarosimon** [comment]
@samary something like this? I have to check this in our testbed

**Comment by samary** [comment]
@alvarosimon Looks good ! Thanks !

File: aii-opennebula/src/main/resources/vmtemplate.tt [hunk 1]
```diff
@@ -89,4 +89,7 @@ PCI = [
 [%- IF system.opennebula.labels.defined %]
 LABELS = "[% system.opennebula.labels.join(',') %]"
 [%- END %]
+[%- FOREACH sched IN system.opennebula.placements %]
+[%          sched.key FILTER upper %] = "[% sched.value %]"
```
Comment (stdweird) [inline aii-opennebula/src/main/resources/vmtemplate.tt]: fix inner indentation

**Comment by stdweird** [comment]
minor remark

**Comment by alvarosimon** [comment]
@stdweird done!

**Comment by alvarosimon** [comment]
It was tested in altaria

**Pull request merged**